mssql-ballerina/Dependencies.toml

```diff
@@ -0,0 +1,26 @@
+[[dependency]]
+org = "ballerina"
+name = "test"
+version = "0.0.0"
+
+[[dependency]]
+org = "ballerina"
+name = "file"
+version = "0.7.0-alpha5"
```

Comment:
Same. This will break if we update versions in gradle.properties

Reply:
Fixed with [`9dfb4e8`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/98fb4e854661be330b7cf90f81bf53ec0b3f522a)